inc/netpoint.class.php

```diff
@@ -1,432 +0,0 @@
-<?php
```

Comment:
Depreciation should probably be handled :/

Reply:
Also, you should add an entry in the changelog ;)

Reply:
done